File: Math/Easy/1822/1822.py
from typing import List

class Solution:
    def arraySign(self, nums: List[int]) -> int:
        negs = 0
        for num in nums:
            if num == 0:
                return 0
            elif num < 0:
                negs += 1
        return -1 if negs % 2 else 1

File: Math/Easy/1822/1822.c
int arraySign(int* nums, int numsSize){
    int negs = 0;
    for(int i = 0; i < numsSize; i++){
        if(nums[i] < 0){
            negs++;
        }
        if(nums[i] == 0){
            return 0;
        }
    }
    return negs % 2 ? -1 : 1;
}